### PR TITLE
Review

### PR DESCRIPTION
**Pull request by swapnith8**
I have completed Review page

### CODE REVIEW

File: create.py [hunk 1]
```diff
@@ -80,4 +84,62 @@ def logout():
...
+        flash(var1)
+        # url="/review/"+str(id)
+        
+        return redirect(url_for('books', id = id))
+
+    else:   
+        return render_template("Book_Page.html", Book_details=result,comments=r, allreviewdata = data )
+
+
+# @app.route('/review/<id>', methods=['POST','GET'])
```
Comment (ashishk144) [inline create.py]: remove commented code.

File: create.py [hunk 2]
```diff
@@ -80,4 +84,62 @@ def logout():
...
+        print(user)
+        if user is not None:
+            print("User had already given rating.")
+            var1 = "Error: User had already given rating."
+            return render_template("Book_Page.html", Book_details=result,var1 = var1,comments=r, allreviewdata = data )
+        db.session.add(reviewdata)
+        db.session.commit()
+        var1="Review submitted"
+        flash(var1)
+        # url="/review/"+str(id)
```
Comment (ashishk144) [inline create.py]: remove commented code

File: templates/Book_Page.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Review</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
```
Comment (ashishk144) [inline templates/Book_Page.html]: extend to a layout file

File: templates/Book_Page.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+
+</head>
+<style>
```
Comment (ashishk144) [inline templates/Book_Page.html]: Use stylesheets for CSS

File: create.py [hunk 5]
```diff
@@ -80,4 +84,62 @@ def logout():
...
+@app.route('/books/<id>',methods=['POST','GET'])
+def books(id):
+    print(id)
+    result = db.session.query(Books).filter(Books.isbn == id).first()
+    print(result)
+    data=Review.query.all()
+    r=Review.query.filter_by(isbn=id).all()
+    print(r)
+    if request.method=='POST':
+        reviewdata=Review(request.form['names'],id,request.form['email'],request.form['comment'],request.form['rating'])
```
Comment (ashishk144) [inline create.py]: data must not be again asked to re enter. Use the session variable to fetch the details of the user.

File: create.py [hunk 6]
```diff
@@ -80,4 +84,62 @@ def logout():
         return render_template("reg.html", var1 = var1)
     except:
         var1 = "You must first log in to logout"
-        return render_template("reg.html",var1 = var1)
\ No newline at end of file
+        return render_template("reg.html",var1 = var1)
+@app.route('/books/<id>',methods=['POST','GET'])
+def books(id):
+    print(id)
```
Comment (ashishk144) [inline create.py]: check if the user is logged in or not before accepting any review.

**Review by ashishk144 — COMMENTED** [review]
Work on the feedback.

File: create.py (hunk 6 above)
Comment (swapnith8) [reply create.py]: updated user is login or not in create.py

File: create.py (hunk 1 above)
Comment (swapnith8) [reply create.py]: updated

File: create.py (hunk 2 above)
Comment (swapnith8) [reply create.py]: updated

File: templates/Book_Page.html (hunk 3 above)
Comment (swapnith8) [reply templates/Book_Page.html]: made changes and commited @ashishk144

File: templates/Book_Page.html (hunk 4 above)
Comment (swapnith8) [reply templates/Book_Page.html]: updated

File: create.py (hunk 5 above)
Comment (swapnith8) [reply create.py]: updated based on feedback @ashishk144

**Comment by ashishk144** [comment]
Looks good to merge.

**Pull request merged**